server/tournaments/index.js

```diff
@@ -83,7 +84,8 @@ class Tournament extends Rooms.RoomGame {
 		this.playerCap = (playerCap ? parseInt(playerCap) : Config.tourdefaultplayercap) || 0;
 
 		/** @type {string} */
-		this.format = formatId;
+		// This will sometimes be sent alone in updates as "format", if the tour doesn't have a custom name
+		this.name = name || formatId;
```

Comment:
In theory we might want name to default to e.g. "OU (with custom rules)" for formats with custom rules.

Reply:
You can't actually create a tournament with custom rules, so that'd have to be done when setting, but I don't think that's a terrible idea.

Reply:
and / or allowing you to start a tournament with custom rules

Reply:
Yeah, that's what I mean, when setting.